TemsApi/src/Domain/Exceptions/ExistingTypeException.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Domain.Exceptions
+{
+    public class ExistingTypeException : Exception
+    {
+        public ExistingTypeException(string message)
```

Comment:
If we switch to using fluent results (from that video), this exception ont be needed anymore